spec/system/dashboard/user_invitations_spec.rb

```diff
@@ -4,14 +4,22 @@
   context 'admin can invite new user' do
     it 'will send an invitation (pending)' do
       admin = create(:user, roles: :admin)
+      account = admin.account
+      location = Pumi::Province.all.first
 
       sign_in(admin)
       visit new_user_invitation_path
 
       fill_in 'user[email]', with: 'bopha@somleng.com'
+      choose('Admin')
+      select location.name_en, from: 'Locations'
```

Comment:
If we can, we shouldn't need to depend on third-party gems in feature/system level spec because it is coupling with the implementation detail.

I would prefer just picking one province and hard code their name here.